packages/third-parties/src/google/ga.tsx

```diff
@@ -54,14 +54,14 @@ export function GoogleAnalytics(props: GAParams) {
   )
 }
 
-export const sendGAEvent = (...args: Object[]) => {
+export function sendGAEvent(..._args: Object[]) {
```

Comment:
Converted to a `function` so that `arguments` would be available. 

Wasn't entirely sure of any other way to turn the `args` into `Arguments()` instead of `Array()`, so I just prefixed it with `_` to have the types at least. Not sure though.